docs/api/public.html

```diff
@@ -19,6 +19,8 @@
   <body>
     <redoc spec-url='https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/public_api_v1.yaml'>
     </redoc>
-    <script src="https://rebilly.github.io/ReDoc/releases/latest/redoc.min.js"> </script>
+  <script src="https://rebilly.github.io/ReDoc/releases/latest/redoc.min.js"
+    integrity="sha384-AllPzUrjbdGnf7PlZwIN0YaHKb/Q1kaAh6ZHToYJAAC2DtsTmez4ga5ZwvW7r9f2" 
```

Comment:
unless we include a specific release (for which the sha384 doesn't change), I'm afraid once they bump the latest/redoc.min.js the hash is going to change and therefore cause troubles.

Reply:
correct, I'll fix this asap